chapter_01.asciidoc

```diff
@@ -73,9 +73,11 @@ from selenium import webdriver
 browser = webdriver.Firefox()
 browser.get("http://localhost:8000")
 
-assert "Django" in browser.title
+assert "Congratulations!" in browser.title
 ----
 ====
+// CSANAD: They changed the default title for Django, it currently says
+//      "The install worked successfully. Congratulations!"
```

Comment:
oh i'm going to have to take a look at that!  there's supposed to be a test that checks for this...